int64 current_index
int64 predicted_index
---
string estimate